src/lsst/cmservice/handlers/functions.py

```diff
@@ -83,7 +83,7 @@ async def upsert_spec_block(
     spec_block = spec_block_result.first()
     if spec_block and not allow_update:
         print(f"SpecBlock {key} already defined, skipping it")
```

Comment:
Given updated return value below, are you still "skipping it"?

Reply:
We are not updating it, which seems like "skipping" it.   I could say "leaving it unchanged".